### ISSUE TITLE
Working after bloomsky shutdown?

### ISSUE THREAD

**Issue by ERisEPR** [mrchapp/florescielo-api#1]
Would this project be for bloomsky hardware to work after bloomsky's shut down?

**Comment by mrchapp** [comment]
Yes, it works as long as there is DNS entry pointing to this deployment.
